KEP Template: Add documentation to template about feature gates (#3069)
kubernetes/enhancements: keps/NNNN-kep-template/README.md

Define graduation milestones.

> These may be defined in terms of API maturity, [feature gate] graduations, or as

Reviewer: Was this meant to be a link?

In general we try to use the same stages (alpha, beta, GA), regardless of how the
functionality is accessed.

> [feature gate]: https://git.k8s.io/community/contributors/devel/sig-architecture/feature-gates.md

Reviewer 2: Aside - we should look to moving that doc into https://www.kubernetes.dev/docs/

Describe the consequences on existing workloads (e.g., if this is a runtime
feature, can it break the existing applications?).

> Feature gates are typically disabled by setting the flag and restarting the
> component. No other changes should be necessary to disable the feature.

Reviewer 2: ```suggestion
Feature gates are typically disabled by setting the flag to `false` and restarting the
```

Author: Oops, good catch

Author: oh oops the suggestion deleted the second half of that line. let me fix manually
